docs/en/ingest-management/agent-policies.asciidoc

```diff
@@ -273,3 +273,9 @@ Make sure that the {agent}s assigned to this policy all have connectivity to the
 that you added. Lack of connectivity will prevent the {agent}
 from checking in with the {fleet-server} and receiving policy updates, but the agents
 will still forward data to the cluster.
+
+[discrete]
+[[agent-policy-scale]]
+== Scale of Policies Supported
+
+Maximum number of Elastic Agent policies supported in a single {fleet} is 500. You may experience slowness in response when a higher number of policies are configured. 
```

Comment:
I have a couple questions before I can edit this part:

* What do we mean by a "single Fleet"? Are we talking about one instance of Kibana with Fleet? My concern here is that users will conflate a "single Fleet" with the number of Fleet Servers they have deployed. Can you think of a better way to describe this?
* How does the user experience the slow response that you describe? Are you talking about UI performance? Or is there some delay in the data being collected, parsed, and indexed into Elasticsearch?

Reply:
@dedemorton I modified it to just say "fleet". Slowness in response is in UI. If you feel that is too much information we can drop it. thanks.